README.md

```diff
@@ -166,10 +166,11 @@ In addition, we use assembly from [s2n-bignum](https://github.com/awslabs/s2n-bi
 | Algorithms | Routines | CPUs |
 | ----------| ------------| ------------|
 | RSA | Montgomery multiplication | aarch64 |
-| P-384 | field operations | aarch64, x86_64 |
-| P-521 | field operations | aarch64, x86_64 |
-| X25519 | field operations, group operations, scalar point multiplication | aarch64, x86_64 |
-| Ed25519 |  encode, decode, scalar point multiplication | aarch64, x86_64 |
+| P-256 | field operations, scalar point multiplication | aarch64, x86_64 |
+| P-384 | field operations, group operations, scalar point multiplication | aarch64, x86_64 |
+| P-521 | field operations, group operations, scalar point multiplication | aarch64, x86_64 |
+| X25519 | scalar point multiplication | aarch64, x86_64 |
+| Ed25519 | field operations, encode, decode, scalar point multiplication | aarch64, x86_64 |
```

Comment:
NP: This table indicates what *IS* covered, but there's no indication for what *IS NOT* covered. 

For EC, the range of "Routines" appears to be:
* field operations
* group operations
* scalar point multiplication
* encode
* decode

Are all 5 Routines above applicable to all 5 EC algorithms? (Does a Routine not being listed for an EC imply that the Routine is not applicable, or that the Routine is applicable but not (yet) formally verified.)

Do these 5 "Routines" cover all that could potentially (and reasonably) be formally verified for an EC?

Reply:
I looked into each function that are provided by s2n-bignum and summarizes them as one of the categories. So these are routines categories that are indeed verified. But I can see that it causes confusion because one couldn't know if there are other functions that are in these routine categories but are not verified. Will it be better if I list the s2n-bignum function names?

Reply:
>Are all 5 Routines above applicable to all 5 EC algorithms? (Does a Routine not being listed for an EC imply that the Routine is not applicable, or that the Routine is applicable but not (yet) formally verified.)
Things get a bit more involved than that because some of the s2n-bignum functions consume another category. For example, the scaler point multiplication could be implemented directly without calling point_add and point_double so not having a routine category doesn't mean it is not applicable nor it is not formally verified.

Reply:
Updated the PR with a new commit that lists out all functions verified through s2n-bignum.